src/Files.App/Views/Shells/BaseShellPage.cs

```diff
@@ -135,9 +135,11 @@ public bool IsCurrentInstance
 					_IsCurrentInstance = value;
 
 					if (!value && SlimContentPage is not ColumnViewBrowser)
+					{
 						ToolbarViewModel.IsEditModeEnabled = false;
-
-					NotifyPropertyChanged(nameof(IsCurrentInstance));
+						ToolbarViewModel.IsCommandPaletteOpen = false;
+						NotifyPropertyChanged(nameof(IsCurrentInstance));
```

Comment:
`NotifyPropertyChanged(nameof(IsCurrentInstance))` was originally located outside of `if`. This should be outside the `if` block.

Reply:
Fixed in the latest commit.